operator/config/crd/patches/webhook_in_alertingrules.yaml

```diff
@@ -0,0 +1,16 @@
+# The following patch enables a conversion webhook for the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: alertingrules.loki.grafana.com
+spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      clientConfig:
+        service:
+          namespace: system
+          name: webhook-service
+          path: /convert
```

Comment:
This a generated file in case we want later to enable a mutating webhook for conversions. This is not referenced anywhere. This is auto-generated TBH. In fact if we use it ever, the namespace will be replace by the kustomization options of each overlay, e.g.:
```
# Adds namespace to all resources.
namespace: openshift-logging
```